test/unit/guard_clause_test.exs

```diff
@@ -136,18 +136,24 @@ defmodule ExFormat.Unit.GuardClauseTest do
        assert_format_string "@spec var(var, context) :: {var, [], context} when var: atom, context: atom\n"
     end
 
-    @tag :skip
     test "when guard clause is on different line as type spec" do
       assert_format_string """
       @spec flat_map_reduce(t, acc, fun) :: {[any], any}
             when fun: (element, acc -> {t, acc} | {:halt, acc}),
                  acc: any
+      """
 
+      assert_format_string """
       @spec send(dest, msg, [option]) :: :ok | :noconnect | :nosuspend
             when dest: pid | port | atom | {atom, node},
                  msg: any,
                  option: :noconnect | :nosuspend
+      """
+    end
 
+    @tag :skip
+    test "when guard clause contains Type.t, it shouldn't be parenthesized" do
+      assert_format_string """
       @spec transform(Enumerable.t, acc, fun) :: Enumerable.t
             when fun: (element, acc -> {Enumerable.t, acc} | {:halt, acc}),
```

Comment:
I think for this case, I skipped the test because it formats `Enumerable.t ` as `Enumerable.t()` inside guard. So I was thinking we should strip parens for all zero arity calls inside guard, if the guard is part of the type spec?